pkgs/by-name/op/opensta/package.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, cmake
+, flex
+, bison
+, swig4
+, fetchFromGitHub
+, fetchurl
+, tcl
+, tclreadline
+, zlib
+}:
+
+let
+  cudd = stdenv.mkDerivation rec {
```

Comment:
can this be a separated package?

Reply:
Yes, would changing this make it ready to merge?

Reply:
After that I wanna test it.